packages/yarnpkg-pnpify/sources/commands/SdkCommand.ts

```diff
@@ -113,6 +121,7 @@ export default class SdkCommand extends Command {
         onlyBase,
         configuration,
         verbose: this.verbose,
+        withPnpify: this.compat,
```

Comment:
@oliversalzburg should be `compat` as well, I think

Reply:
How far should I pass it through as `compat`? In the final usage in the `TEMPLATE`, the name was `usePnpify`. At some point I will have to transition.

Reply:
@oliversalzburg "How far should I pass it through as `compat`". I think it should be "compat" everywhere, except in TEMPLATE it should be "usePnpify"